### PR TITLE
Add support for Parblo Ninos N4

### PR DESCRIPTION
**Pull request by cantgetin**

### CODE REVIEW

**Review by gonX — APPROVED** [review]

File: OpenTabletDriver.Configurations/Configurations/Parblo/Ninos N4.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "Name": "Parblo Ninos N4",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 101.0,
+      "Height": 76.2,
+      "MaxX": 21596.0,
+      "MaxY": 16198.0
```
Comment (Kuuuube) [inline OpenTabletDriver.Configurations/Configurations/Parblo/Ninos N4.json:8]: This tablet appears to be 5080 lpi so the Width and Height should be 107.98 x 80.99 mm if the MaxX and MaxY are accurate. Double check that you cant get any higher for those.

Not having round numbers is fine if the tablet actually reports that but most tablets do have round numbers. Make sure you cant get 22000 x 16200 for MaxX and MaxY. Or if you can, it should be updated. (Width and Height would be 108 x 81 mm if they are round)

Comment (cantgetin) [reply OpenTabletDriver.Configurations/Configurations/Parblo/Ninos N4.json:8]: It was a bit tricky but I did get 21600 horizontally and 16200 vertically in debugger. I'll update that. Can't go higher than 21600 horizontally though. Do you need a video proof or some recorded data? 

Will update Width and Height. If it's 21600 MaxX and 16200 MaxY then the Width should be 21600/5080 = 4.2519685 inches which is 108mm and Height should be 16200/5080 = 3.18897637795 inches which is 81mm.

Comment (Kuuuube) [reply OpenTabletDriver.Configurations/Configurations/Parblo/Ninos N4.json]: Your verification is all that's needed. Looks good.

**Review by X9VoiD — COMMENTED** [review]
This tablet has no buttons anywhere? Even on the pen itself?

**Comment by X9VoiD** [comment]
Did my research and yes, no buttons at all

https://www.parblo.com/products/ninos-n4-n7-n7b-drawing-tablet?variant=40901972394031
image

**Review by X9VoiD — APPROVED** [review]

**Comment by X9VoiD** [comment]
@InfinityGhost I can't merge, this PR is from a first-time contributor so Github CI didn't run but this PR should be good to merge.

**Review by InfinityGhost — APPROVED** [review]

**Pull request merged**